### ISSUE TITLE
History Previous & Next shortcuts does not work on macOS with Alt key

### ISSUE THREAD

**Issue by Dentrax** [godotengine/godot#44858]
**Godot version:**
`3.2.3.stable`

**OS/device including version:**
`macOS 11.0.1`

**Issue description:**
<kbd>Alt+Left</kbd>
<kbd>Alt+Right</kbd>
...keys not works as expected in Editor window.

**Steps to reproduce:**
1. Type <kbd>Alt+Left</kbd> here
<img width="169" alt="Screen Shot 2021-01-01 at 22 18 30" src="https://user-images.githubusercontent.com/16493751/103444954-49f0af80-4c7f-11eb-9a10-dfe674ed5dc9.png">

2. See the cursor that moves end of the current word.
<img width="166" alt="Screen Shot 2021-01-01 at 22 19 39" src="https://user-images.githubusercontent.com/16493751/103444963-7278a980-4c7f-11eb-80a8-6b3b604ee590.png">

My Config:

`File > History ....`

<img width="296" alt="Screen Shot 2021-01-01 at 22 16 23" src="https://user-images.githubusercontent.com/16493751/103444929-fd0cd900-4c7e-11eb-8b6b-298148292312.png">

If I click here, it works.

**Expected Behaviour**

It should return previous window from the histroy.

**Comment by EricEzaM** [comment]
Yes, this conflicts with mac shortcuts text editor shortcuts defined by Apple themselves. 

> - Option–Left Arrow: Move the insertion point to the beginning of the previous word.
> - Option–Right Arrow: Move the insertion point to the end of the next word.

https://support.apple.com/en-us/HT201236

It's probably best to not conflict with those. However, this could potentially be solved by #43663.

**Comment by Dentrax** [comment]
In this case, we can change default keybinding to;

<kbd>Option+CMD+Left</kbd>
<kbd>Option+CMD+Right</kbd>

So this should not break default editor compatibility keybinding for macOS devices.

**Comment by Dentrax** [comment]
ping @akien-mga @reduz

**Comment by Calinou** [comment]
@Dentrax Please do not ping engine maintainers about an issue they don't know how to fix :slightly_smiling_face:
